src/main/php/PDepend/Report/Jdepend/Chart.php

```diff
@@ -138,6 +138,7 @@ public function log(Analyzer $analyzer)
      * Closes the logger process and writes the output file.
      *
      * @throws NoLogOutputException If the no log target exists.
+     * @throws RuntimeException
```

Comment:
```suggestion
     * @throws RuntimeException If a layer element is missing.
```

Would be nice to have some explanation.
Otherwise, we have to have discuss about useless doc blocks again. 😏 

Applies to all of these cases.

Reply:
See https://github.com/pdepend/pdepend/pull/816